test-openvpn.ps1

```diff
@@ -35,12 +35,16 @@ Function Verify-Path ([string]$mypath) {
     }
 }
 
+$exe_path = (Get-ItemProperty 'HKLM:\SOFTWARE\OpenVPN' -Name 'exe_path').exe_path
+
 if (! $Openvpn) {
-    $Openvpn = "C:\Program Files\OpenVPN\bin\openvpn.exe"
+    $Openvpn = $exe_path
 }
 
 if (! $Gui) {
-    $Gui = "C:\Program Files\OpenVPN\bin\openvpn-gui.exe"
+    # Assume openvpn-gui.exe is in the same directory as openvpn.exe
+    $exe_dir = $exe_path.TrimEnd('openvpn.exe')
+    $Gui = "${exe_dir}openvpn-gui.exe"
```

Comment:
I'd use JOin-Path for that.

like 

$Gui = (Join-Path $exe_dir 'openvpn-gui.exe')

it is very rare case, Join-Path takes care of path separator, which is different on Mono and Windows.
I wouldn't beleive we'll be running openvpn on Linux/Mono sometimes :)

Join-Path is considered a safe way of joining paths together